phoenix-core/src/main/java/org/apache/phoenix/iterate/ChunkedResultIterator.java

```diff
@@ -56,6 +57,7 @@
     private final MutationState mutationState;
     private Scan scan;
     private PeekingResultIterator resultIterator;
+    private QueryPlan plan;
 
     public static class ChunkedResultIteratorFactory implements ParallelIteratorFactory {
```

Comment:
Currently removed the changes required for ChunkedResultIterator. The changes in the chunked result iterator are just removing code not required.